packages/core/src/components/ui/VirtualTable/_VirtualTable.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../../common/styles/colors";
+
+.ReactVirtualized__Table__headerRow {
+  border-bottom: none;
+  padding-right: 0 !important;
+  height: 48px;
+}
+
+.ReactVirtualized__Table__headerColumn {
+  font-weight: normal;
+
+  > * {
+    font-size: 11px !important;
+  }
+}
+
+.ReactVirtualized__Table__headerTruncatedText {
+  margin-right: 4px;
+  font-weight: bold;
+  font-size: 13px;
+  text-transform: none;
+}
+
+.ReactVirtualized__Table__row {
+  color: $white;
+
+  > * {
+    font-size: 13px;
+    line-height: 20px !important;
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.vt-container {
+  display: flex;
+
+  .hfui-table {
+    flex: 1 1 auto;
+    height: calc(100vh - 35px);
+    width: 100vw;
+  
+    .icons-cell > * {
+      margin-right: 8px;
+  
+      &:last-child {
+        margin-right: 0;
+      }
+  
+      &:hover {
+        cursor: pointer;
+        color: $yellow1;
```

Comment:
seems yellow wont match with light theme of ufx-ui, can you keep it to similar to hover style of [_Table.scss](https://github.com/bitfinexcom/ufx-ui/blob/staging/packages/core/src/components/ui/Table/_Table.scss)